docs/analyzer.md

```diff
@@ -16,9 +16,10 @@ PM> Install-Package FakeItEasy.Analyzer
 
 ## Diagnostics
 
-The analyzer currently provides two diagnostics:
+The analyzer currently provides the following diagnostics:
 
 | Id             | Summary                          | Description                                                                            |
 |----------------|----------------------------------|----------------------------------------------------------------------------------------|
 | FakeItEasy0001 | Unused call specification        | Triggered when you specify a call but don't configure or assert it, making it a no-op. |
 | FakeItEasy0002 | Non-virtual member configuration | Triggered when you try to configure a non-virtual member, which cannot be faked.       |
+| FakeItEasy0003 | Argument constraint outside call specification | Triggered when you try to use an argument constraint (`A<T>.Ignored`, `A<T>.That...`) outside a call specification (`A.CallTo`, `Fake<T>.CallsTo`) |

```

Comment:
aside from my comment about adding links to the other topics, how do you feel about a `&hellip;` instead of `...`? No pressure!